util/measures/2019/enriched-measures-data-quality.json

```diff
@@ -2488,7 +2488,7 @@
     }
   },
   {
-    "title": "Colonoscopy Interval for Patients with a History of Adenomatous Polyps \r- Avoidance of Inappropriate Use",
+    "title": "Colonoscopy Interval for Patients with a History of Adenomatous Polyps\\r\\n \r- Avoidance of Inappropriate Use",
```

Comment:
@lamroger Same issue from the csv adding in `\\r\\n` here. Is this something we need to fix?

Reply:
Seems like somethign might need to fix. Those single \r's also look wrong...

Reply:
@KyleApfel @harrisj It looks like it was an issue with `util/measures/2019/quality-measures.csv`. After removing the additional `\r\n` from this line in that file, it reverted back to only having `\r-` only. I am hesitant on changing this as this is how it is for the 2018 version of the measure data.